### ISSUE TITLE
the pretrained weight for PH2 dataset of BAT(paper)

### ISSUE THREAD

**Issue by fengjiejiejiejie** [jcwang123/BA-Transformer#9]
Hi, 
Could you please provide the pretrianed weight for PH2 dataset of BAT(paper)?
BAT (paper) | 0.921 | 0.858 | - | -
Thanks very much!

Best,
Fj.

**Comment by fengjiejiejiejie** [comment]
Hi,
With the given pretrained weight (bat_valid_1_1_0_e6_loss_0_aug_1), the test jac and dice are 0.847 and 0.914, respectively.
Are these results are right?
Best, 
Fj.

**Comment by jcwang123** [comment]
It seems different with our presented results, BAT(val)

**Comment by fengjiejiejiejie** [comment]
> It seems different with our presented results, BAT(val)
But with the given pretrained weight (bat_1_1_0_e6_loss_0_aug_1), the resutls are the same as you report.

Average dice value of evaluation dataset =  0.9239621061621994
Average jc value of evaluation dataset =  0.8641284064586956
Average hd95 value of evaluation dataset =  7.515767869308079
Average assd value of evaluation dataset =  1.0433042311367364

**Comment by jcwang123** [comment]
Yes, the pre-trained weight presented here is the fine-tuned version after paper.

**Comment by AIddddddd** [comment]
> > It seems different with our presented results, BAT(val)
> > But with the given pretrained weight (bat_1_1_0_e6_loss_0_aug_1), the resutls are the same as you report.
> 
> Average dice value of evaluation dataset = 0.9239621061621994 Average jc value of evaluation dataset = 0.8641284064586956 Average hd95 value of evaluation dataset = 7.515767869308079 Average assd value of evaluation dataset = 1.0433042311367364

hello bro，i use the given pretrained weight (bat_1_1_0_e6_loss_0_aug_1)，but the results are not good as your results
and i use isic2016_pair1_testdata(379nums) 、isic2016_part1_test_truth(379 nums)， Is this dataset the same as yours？
look forward for your reply
my results are follow:
Average dice value of evaluation dataset =  0.7365860132578288
Average jc value of evaluation dataset =  0.6413009585751983
Average hd95 value of evaluation dataset =  25.0441014385714
Average assd value of evaluation dataset =  4.535040351020787

**Comment by wanwan-na** [comment]
Hello, I used the same data set as you and got the same effect. Do you know the reason now?
